ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/validation/CustomValidators.java

```diff
@@ -95,9 +95,10 @@
   private final Class<? extends Statement> statementClass;
   private final StatementValidator validator;
 
-  CustomValidators(
-      final Class<? extends Statement> statementClass,
-      final StatementValidator validator) {
+  <T extends Statement> CustomValidators(
```

Comment:
I tried so hard to get this to work! I didn't think of adding type inference in the constructor 🤦‍♂ well done 🙇